src/Microcosm.js

```diff
@@ -55,11 +55,17 @@ Microcosm.prototype = {
     return this
   },
 
+  shouldTransactionMerge() {
+    return true
+  },
```

Comment:
just a note that your example (I believe) will just save the first 500 transactions, and then merge the rest in. As apposed to saving the last 500 (which would probably involve merging the early transactions into each other).